### PR TITLE
Update addon layout according to latest ember addon blueprint

### PR DESCRIPTION
**Pull request by pichfl**
- New run scripts
- Update eslint configuration

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,11 +14,16 @@
...
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint test:*",
```
Comment (Turbo87) [inline package.json:24]: I'd prefer to keep `yarn test === ember test`. Having `yarn test` automatically start ember-try always causes unintentional pain 😩

File: package.json [hunk 2]
```diff
@@ -39,21 +45,23 @@
...
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-ember": "^10.2.0",
     "eslint-plugin-import-helpers": "^1.1.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.3.1",
     "eslint-plugin-promise": "^4.2.1",
     "loader.js": "^4.7.0",
+    "npm-run-all": "^4.1.5",
     "prettier": "^2.2.1",
+    "qunit": "^2.14.0",
```
Comment (Turbo87) [inline package.json:64]: v2.14 has significant UI changes that causes the fixture container to unintentionally overlap the main UI. I'd prefer to not us that version yet.

File: package.json (hunk 1 above)
Comment (pichfl) [reply package.json:24]: I try to adhere to the recommended addon layout, but I think I can agree with that. Will update.

File: package.json (hunk 2 above)
Comment (pichfl) [reply package.json]: Fine with me.

**Review by Turbo87 — APPROVED** [review]

**Pull request merged**